scripts/build_script.sh

```diff
@@ -23,7 +24,7 @@ GENERATE_COVERAGE=0
 GENERATE_DEBIAN_PACKAGE=0
 
 # Get the options
-while getopts "cdhps:" option; do
+while getopts "cdhpsD:" option; do
```

Comment:
Is this case sensitive? How about we use consistent case here: all lower or all upper? Is it possible to make this option case insensitive?